src/markdown-pages/docs/writing-own-tests/running-test-modes.md

```diff
@@ -35,21 +40,32 @@ You can run a test in this mode by using the `runs submit local` <a href="https:
 
 ## <a name="hybrid"></a>Running a test locally but using shared configuration
 
-When you run a test locally, but using shared configuration, the Galasa bootstrap is set to the URL of the Galasa Ecosystem where the shared configuration is stored. The Galasa framework is launched within the JVM on the local machine, but the framework consults the remote ecosystem to read configuration data, but not the credentials properties as these are drawn from a local file. This is the key difference between running a test in this "hybrid" mode versus running a test locally without using shared configuration. In hybrid mode, the test still runs in the local JVM and all test results and artifacts are stored on the local disk. 
+When you run a test locally, but using shared configuration, the Galasa bootstrap is set to the URL of the Galasa Ecosystem where the shared configuration is stored. The Galasa framework is launched within the JVM on the local machine, but the framework consults the remote Ecosystem to read configuration data, but not the credentials properties as these are drawn from a local file. This is the key difference between running a test in this "hybrid" mode versus running a test locally without using shared configuration. In hybrid mode, the test still runs in the local JVM and all test results and artifacts are stored on the local disk. 
 
 
 ![running in local mode with shared configuration:](hybridrunmode.svg)
 
-You can run a test in this mode by setting up your bootstrap to refer to the ecosystem in which the shared configuration is stored, and using the `runs submit local` <a href="https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl_runs_submit_local.md" target="_blank"> Galasa CLI</a> command. 
+To run tests in hybrid mode, add the following properties into your local `bootstrap.properties` file:
+
+```
+framework.config.store=galasacps://my.ecosystem.url/api
+framework.extra.bundles=dev.galasa.cps.rest
+```
+
+where: <br>
+- `https://my.ecosystem.url` refers to the Web UI used to allocate tokens and <br>
+- `framework.extra.bundles` tells the Galasa framework to load the `dev.galasa.cps.rest` extension. This extension tells the Galasa framework how to handle URLs that start with `galasacps` as the *scheme* part of the URL. <br>
+
+After setting the `GALASA_TOKEN` to be a valid token for the Ecosystem from which the CPS property values will be drawn, you can run a test in hybrid mode by setting your bootstrap to refer to the Ecosystem in which the shared configuration is stored, and using the `galasactl runs submit local` <a href="https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl_runs_submit_local.md" target="_blank"> Galasa CLI</a> command. 
```

Comment:
> After setting the `GALASA_TOKEN` to be a valid token for the Ecosystem from which the CPS property values will be drawn, 

... log into the Ecosystem using `galasactl auth login`

Then 

> you can run a test in hybrid mode by setting your bootstrap to refer to the Ecosystem in which the shared configuration is stored,

Reply:
updated